src/core_atmosphere/Registry.xml

```diff
@@ -2609,6 +2627,17 @@
                      description="EDMF detrainment of water vapor mixing ratio in the MYNN PBL scheme"
                      packages="bl_mynn_in"/>
 
+		 <var name="maxmf" type="real" dimensions="nCells Time" units="m s^{-1}"
+                     description="Maximum mass flux in the each column"
+                     packages="bl_mynn_in"/>
+
+                <var name="maxwidth" type="real" dimensions="nCells Time" units="m"
+                     description="Maximum width of updrafts"
+                     packages="bl_mynn_in"/>
+
+                <var name="ztop_plume" type="real" dimensions="nCells Time" units="m"
+                     description="Height of highest penetratign plume"
```

Comment:
@joeolson42 typo here: description="Height of highest penetratign plume"

Reply:
```suggestion
                     description="Height of highest penetrating plume"
```